File: CS/AccentInsensitiveEditSettings.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.Xpf.Grid.LookUp;

namespace AccentInsensitiveSearch {
    public class AccentInsensitiveEditSettings : LookUpEditSettings {
        protected override DevExpress.Xpf.Editors.Helpers.ItemsProvider CreateItemsProvider() {
            return new AccentInsensitiveItemsProvider(this);
        }
    }
}

File: VB/AccentInsensitiveEditSettings.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.Xpf.Grid.LookUp

Namespace AccentInsensitiveSearch
	Public Class AccentInsensitiveEditSettings
		Inherits LookUpEditSettings
		Protected Overrides Function CreateItemsProvider() As DevExpress.Xpf.Editors.Helpers.ItemsProvider
			Return New AccentInsensitiveItemsProvider(Me)
		End Function
	End Class
End Namespace
